python/servo/platform/base.py

```diff
@@ -112,6 +113,16 @@ def install_taplo(self, force: bool) -> bool:
 
         return True
 
+    def install_crown(self, force: bool) -> bool:
+        if not force and shutil.which("crown") is not None:
+            return False
+
+        if subprocess.call(["cargo", "install", "crown", "--git", "https://github.com/sagudev/cargo-crown"],
```

Comment:
I would prefer if the plugin were maintained in-tree

Reply:
I though about this before, but I just see crown also as linter for other mozjs consumer. Although while crown is implemented using internal rustc stuff, it is locked to dame rustc version as servo (+/- some versions until it breaks), so maybe it's not time for it yet.

And we also need to keep rust-toolchain in bringing it back into tree seems more right. It was easier to get it running as separate entity though.